shared/people/index.shared.tsx

```diff
@@ -7,10 +6,13 @@ import Todo from './todo/container'
 import FollowNotification from './follow-notification'
 import Announcement from './announcement/container'
 import FollowSuggestions from './follow-suggestions'
-import {type Props} from '.'
+import {Props} from '.'
 import AirdropBanner from '../wallets/airdrop/banner/container'
 
-export const itemToComponent: (Types.PeopleScreenItem, Props) => React.Node = (item, props) => {
+export const itemToComponent: (arg0: Types.PeopleScreenItem, props: Props) => React.ReactNode = (
```

Comment:
Why `arg0` and not `item`?

Reply:
That was from the converter, I'll change